litgpt/args.py

```diff
@@ -50,24 +50,24 @@ def __post_init__(self) -> None:
                 "`--train.lr_warmup_steps` should be less than `--train.max_steps`."
                 f" Got {self.lr_warmup_steps} lr_warmup_steps and {self.max_steps} max_steps.", UserWarning)
 
-    def gradient_accumulation_iters(self, devices: int) -> int:
+    def gradient_accumulation_iters(self, world_size: int) -> int:
         """Number of iterations between gradient synchronizations"""
-        gradient_accumulation_iters = self.batch_size(devices) // self.micro_batch_size
+        gradient_accumulation_iters = self.batch_size(world_size) // self.micro_batch_size
         assert gradient_accumulation_iters > 0
         return gradient_accumulation_iters
 
-    def batch_size(self, devices: int) -> int:
+    def batch_size(self, world_size: int) -> int:
         """Number of samples between optimizer steps per data-parallel rank"""
-        batch_size = self.global_batch_size // devices
+        batch_size = self.global_batch_size // world_size
```

Comment:
Thanks for pointing to the discussion regarding keeping the previous behavior. I'll revert all changes related to `TrainArgs` and their method calls, then?

Reply:
Yes, reverting the changes to the batch size related methods in args.py and calls in the script would be great, but it might be a lot of work to do manually, sorry. 

I think adding num_nodes is great. Setting the env var is not meant to be done by the user but by the launcher, so I would not do that.

Reply:
Sounds good, thanks for the feedback!

Reply:
I removed it all, PTAL.

Reply:
Thanks for making the adjustments 🎉